src/player.js

```diff
@@ -399,6 +407,47 @@ export default class Player extends FakeEventTarget {
    * @private
    */
   _aspectRatio: ?string;
+  /**
+   * An intersection observer instance to track the player visibility.
+   * @type {IntersectionObserver}
+   * @private
+   */
+  _intersectionObserver: window.IntersectionObserver;
+
+  /**
+   * Whether the player is visible in the page scroll view
+   * @type {boolean}
+   * @private
+   */
+  _isVisibleInScroll: boolean;
+
+  /**
+   * Whether the player browser tab is active or not
+   * @type {boolean}
+   * @private
+   */
+  _isTabVisible: boolean;
+
+  /**
+   * Whether the player browser tab is active and in the scroll view
+   * @type {boolean}
+   * @private
+   */
+  _isVisible: boolean;
```

Comment:
is not `_isVisibleInScroll && _isTabVisible`?

Reply:
not relevant anymore